src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Architecture.cs

```diff
@@ -50,5 +50,20 @@ public enum Architecture
         /// A PowerPC 64-bit (little-endian) processor architecture.
         /// </summary>
         Ppc64le,
+        /// <summary>
+        /// RISCV32
+        /// </summary>
+        Riscv32,
+        /// <summary>
+        /// RISCV64
+        /// </summary>
+        /// <remarks>
+        /// This value indicates (TODO: write here what RV64 ISA we implemet).
```

Comment:
> TODO: write here what RV64 ISA we implemet

I suppose that'd be the standard profile 'RV64GC' (which major distros are currently supporting)?

Not sure if we want 32 (embedded) and 128 (futuristic) variants at this point, since we do not have build support for these variants. (though in src/mono, there is some code for embedded variant, but it's incomplete)